compiler/q-implant/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+nnas_find_package(Jsoncpp)
+if(NOT Jsoncpp_FOUND)
+  message(STATUS "Build jsoncpp: FAILED (missing jsoncpp)")
+  return()
+endif(NOT Jsoncpp_FOUND)
+
+nnas_find_package(LibnpySource QUIET)
+if(NOT LibnpySource_FOUND)
+  message(STATUS "LibnpySource NOT FOUND")
```

Comment:
```suggestion
  message(STATUS "q-implant: LibnpySource NOT FOUND")
```

Reply:
I understood that the way you suggested explicitly leaving a message increases readability.